### PR TITLE
Vector add n

### PR DESCRIPTION
**Pull request by kyleschultz**
Check my awesome function.

### CODE REVIEW

File: math/FunMath.cpp [hunk 1]
```diff
@@ -1 +1,13 @@
 #include "FunMath.h"
+
+//Kyle Schultz, Mitch Massey, Tyler Albee, Liz Parker
+
```
Comment (muzny) [inline math/FunMath.cpp]: Add a function comment that says what this function does, what the parameters/return values are

Comment (muzny) [reply math/FunMath.cpp]: You can also remove your names from this file.

File: math/test.cpp [hunk 2]
```diff
@@ -10,3 +10,10 @@ TEST_CASE("plus operator on ints", "[plus]") {
 }
 
 // Add more test cases down here
+std::vector<int> in = {1, 2, 3, 4, 5};
+std::vector<int> out = {3, 4, 5, 6, 7};
+std::vector<int> out2 = {5, 6, 7, 8, 9};
```
Comment (muzny) [inline math/test.cpp]: Move the vector declarations into the TEST_CASE (avoid global variables!)

**Comment by elpa8934** [comment]
Thank you, Kyle!

**Pull request merged**